libsolidity/ast/AST.h

```diff
@@ -167,9 +167,12 @@ class SourceUnit: public ASTNode, public ScopeOpener
 		int64_t _id,
 		SourceLocation const& _location,
 		std::optional<std::string> _licenseString,
-		std::vector<ASTPointer<ASTNode>> _nodes
+		std::vector<ASTPointer<ASTNode>> _nodes,
+		bool _experimentalParsingEnabled
```

Comment:
We could even do
```suggestion
		bool _experimentalParsingEnabled = false
```

But then again, if I just saw it correctly, that'd just spare us a single change in the tests, so also doesn't really matter.